kv/db_test.go

```diff
@@ -130,14 +130,15 @@ func TestKVDBInternalMethods(t *testing.T) {
 		args  proto.Request
 		reply proto.Response
 	}{
-		{&proto.InternalRangeLookupRequest{}, &proto.InternalRangeLookupResponse{}},
-		{&proto.InternalGCRequest{}, &proto.InternalGCResponse{}},
-		{&proto.InternalHeartbeatTxnRequest{}, &proto.InternalHeartbeatTxnResponse{}},
-		{&proto.InternalPushTxnRequest{}, &proto.InternalPushTxnResponse{}},
-		{&proto.InternalResolveIntentRequest{}, &proto.InternalResolveIntentResponse{}},
-		{&proto.InternalResolveIntentRangeRequest{}, &proto.InternalResolveIntentRangeResponse{}},
-		{&proto.InternalMergeRequest{}, &proto.InternalMergeResponse{}},
-		{&proto.InternalTruncateLogRequest{}, &proto.InternalTruncateLogResponse{}},
+		{&proto.HeartbeatTxnRequest{}, &proto.HeartbeatTxnResponse{}},
```

Comment:
That's just the thing: they're not accessible through HTTP directly, but with this change they *are* accessible via HTTP if you wrap them in a batch. The `allPublicMethods` map in kv/db.go is no longer doing anything useful and we might as well treat all KV methods equivalently (which for now means exposing them all, and soon means removing them all so that only the sql interface is exposed by HTTP).